### PR TITLE
AMO text zh-CN translate (new)

### PR DESCRIPTION
**Pull request by Cye3s**

### CODE REVIEW

File: src/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -439,6 +439,25 @@
     "hash": "2aff10770796694827ec4253860e0771"
   },
 
+  "translatorCredit": {
+    "message": "该扩展简体中文翻译由 Cye3s 完成",
```
Comment (rugk) [inline src/_locales/zh_CN/messages.json]: You know you need to mention the string `$TRANSLATORS$` here?
See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Retrieving_message_strings_from_JavaScript for a different example.

This is the placeholder, that will automatically be replaced. So don't hardcode your name here, this is what we have `translatorUsername` for (and `translatorLink` for the link).

Generally, you can easily test [the add-on locally](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#coding) to see if everything is correctly translated. (You may also notice missing translations then.)

File: src/_locales/zh_CN/messages.json [hunk 2]
```diff
@@ -439,6 +439,25 @@
...
+  },
+  "translatorLink": {
+    "message": "https://github.com/Cye3s",
+    "description": "The link to the translator's GitHub profile."
+  },
+  "translatorUsername": {
+    "message": "Cye3s",
+    "description": "The username that the translator wants to be referred to."
+  },
+
```
Comment (rugk) [inline src/_locales/zh_CN/messages.json]: Apropos missing translations, I think `contributorsThanks` (and similar) are also missing here.

Also remember you can use [the online translator tool](https://lusito.github.io/web-ext-translator/?gh=https://github.com/Cye3s/offline-qr-code), even if you just want to see which translations are missing.

**Comment by Cye3s** [comment]
Sorry, Github newbie, still learning. I will create a branch next time. 
Fix and add "contributors***" string

**Comment by rugk** [comment]
No problem, that's what reviews/my tips are there for. :smiley: 

So I guess it LGTM now. I'll merge it soon… :smile: 

Meanwhile, my obligatory tip that you _can_ translate the wiki, too (it's basically just two FAQ questions), if you want. https://github.com/rugk/offline-qr-code/wiki https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-wiki

**Pull request merged**